das_api/src/builder.rs

```diff
@@ -80,6 +80,12 @@ impl RpcApiBuilder {
             rpc_context.search_assets(payload).await.map_err(Into::into)
         })?;
         module.register_alias("searchAssets", "search_assets")?;
+        
+        module.register_async_method("get_signatures_for_asset", |rpc_params, rpc_context| async move {
+            let payload = rpc_params.parse::<GetSignaturesForAsset>()?;
+            rpc_context.get_signatures_for_asset(payload).await.map_err(Into::into)
+        })?;
+        module.register_alias("getSignaturesForAsset", "get_signatures_for_asset")?;
```

Comment:
Would it make sense to named this as `getAssetSignatures`? We have a `getAssetProof` so I wonder if this one fits the same pattern.

Reply:
We can add an alias in a second PR if people prefer. I wanted to keep it consistent with Solana RPC ("getSignaturesForAddress").